sleap/instance.py

```diff
@@ -333,6 +333,9 @@ def matches(self, other: "Track"):
         """
         return attr.asdict(self) == attr.asdict(other)
 
+    def __hash__(self) -> int:
+        return hash((self.spawned_on, self.name))
+
```

Comment:
Danger zone: let's not override whatever attrs is doing with hashing. [See these docs for reference](https://www.attrs.org/en/stable/hashing.html).

We currently have this class set to `eq=False`, which means it's hashing by object identity. The equivalent of what's being done here would be to set `frozen=True`, which will hash by value. But as you can see above, we have a `matches` method that does basically that, which suggests to me that some other part of the code probably expects it to be hashable by identity instead.